source/auditing-logging.txt

```diff
@@ -133,24 +137,23 @@ Recommendations for Programmatic Access to Audit Logs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To integrate with tools beyond the built-in integrations, we recommend
-that you retrieve logs with the following programmatic tools and feed the JSON-formatted output to your external tools:
-
-- If you use |aws|, you can continually push logs to an |aws| S3
-  bucket. To learn more, see the
-  :oas-atlas-tag:`Push-Based Log Export <Push-Based-Log-Export>` {+atlas-admin-api+} endpoints.
-- Use the {+atlas-admin-api+} endpoints for :oas-atlas-tag:`Logs <MongoDB-Cloud-Users>` and :oas-atlas-tag:`Project and Organization Events <Events>` to
-  retrieve deployment logs and lists of project events.
-- Use the ``atlas deployments logs`` command in the {+atlas-cli+}
-  to retrieve deployment logs. To learn more,
-  see :atlas:`Atlas Deployment Logs 
-  </cli/current/command/atlas-deployments-logs/>`.
+that you retrieve logs with the following programmatic tools and feed
+the JSON-formatted output to your external tools:
+
+- To continually push logs to an |aws| |s3| bucket, use the {+atlas-admin-api+}
+  endpoints for :oas-atlas-tag:`Push-Based Log Export <Push-Based-Log-Export>`.
+- To retrieve deployment logs and lists of project events,
+  use the {+atlas-admin-api+} endpoints for :oas-atlas-tag:`Logs <MongoDB-Cloud-Users>`
+  and :oas-atlas-tag:`Project and Organization Events <Events>`.
+- To retrieve deployment logs, use :ref:`atlas deployment logs <atlas-deployments-logs>`
+  command in the {+atlas-cli+}. To learn more, see :atlas:`Atlas Deployment Logs </cli/current/command/atlas-deployments-logs/>`.
 
 Examples
 --------
 
 .. include:: /includes/complete-examples.rst
 
-The following examples show how to download logs and enable auditing
+The following examples show how to retrieve and download logs and configure auditing
```

Comment:
Can we also please include a link to the TF docs for more info on audit filter? Under the TF example, not here